nixos/modules/services/video/epgstation/default.nix

```diff
@@ -87,6 +87,13 @@ in
       description = "epgstation package to use";
     };
 
+    ffmpeg = lib.mkOption {
+      default = pkgs.ffmpeg-full;
```

Comment:
Does it require ffmpeg-full or could it do with e.g. ffmpeg-headless

Reply:
Practically, it's required for performance reasons. EPGStation streams videos, so it needs hardware accelerated encoding to get acceptable performance. ffmpeg-headless drops support for those, which is a deal breaker for this use case.

Reply:
Hardware acceleration usually comes via VAAPI or VDPAU/NVENC/NVDEC, all of which are support by `ffmpeg-headless`, so I'm not sure what you are missing.

Reply:
Sorry, I missed that. Last time I checked, ffmpeg-headless didn't support libmfx and I misremembered that as not having support for hardware acceleration in general.

I'll fix it in https://github.com/NixOS/nixpkgs/pull/233053.

Reply:
This should use mkPackageOption